### PR TITLE
PLAT-143564: Update screenshot tests for Checkbox and CheckboxItem

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated screenshot tests for Checkbox and CheckboxItem
Checkbox: screenshot tests using `indeterminat` prop
CheckboxItem: screenshot tests using `label`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-143564

### CODE REVIEW

File: tests/screenshot/apps/components/Checkbox.js [hunk 1]
```diff
@@ -3,7 +3,10 @@ import Checkbox from '../../../../Checkbox';
 const CheckboxTests = [
 	<Checkbox />,
 	<Checkbox selected />,
+	<Checkbox selected disabled />,
 	<Checkbox>star</Checkbox>,
-	<Checkbox selected>star</Checkbox>
+	<Checkbox selected>star</Checkbox>,
+	<Checkbox indeterminate />,
+	<Checkbox indeterminate disabled />
 ];
```
Comment (stanca-pop-lgp) [inline tests/screenshot/apps/components/Checkbox.js:11]: Please add some tests for focused checkbox.

Comment (stanca-pop-lgp) [reply tests/screenshot/apps/components/Checkbox.js:11]: Also with indeterminateIcon.

File: tests/screenshot/apps/components/CheckboxItem.js [hunk 2]
```diff
@@ -26,19 +30,25 @@ const CheckboxItemTests = [
```
Comment (stanca-pop-lgp) [inline tests/screenshot/apps/components/CheckboxItem.js:30]: Please add a test with a different icon prop.

Comment (stanca-pop-lgp) [reply tests/screenshot/apps/components/CheckboxItem.js:30]: And some tests with different labelPositions.

File: tests/screenshot/apps/components/CheckboxItem.js [hunk 3]
```diff
@@ -26,19 +30,25 @@ const CheckboxItemTests = [
...
+		<CheckboxItem centered label="label">Hello CheckboxItem</CheckboxItem>,
+		<CheckboxItem centered>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed aliquam dapibus imperdiet. Morbi diam ex, vulputate eget luctus eu, gravida at ligula. Sed tristique eros sit amet iaculis varius. Phasellus rutrum augue id nulla consectetur, a vulputate velit dictum. Vestibulum ultrices tellus ac cursus condimentum. Aliquam sit amet consectetur nulla, viverra bibendum metus.</CheckboxItem>,
+		<CheckboxItem centered label="Really looooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooong label to test centered CheckboxItem with label on RTL locale">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed aliquam dapibus imperdiet. Morbi diam ex, vulputate eget luctus eu, gravida at ligula. Sed tristique eros sit amet iaculis varius. Phasellus rutrum augue id nulla consectetur, a vulputate velit dictum. Vestibulum ultrices tellus ac cursus condimentum. Aliquam sit amet consectetur nulla, viverra bibendum metus.</CheckboxItem>
 	]),
```
Comment (stanca-pop-lgp) [inline tests/screenshot/apps/components/CheckboxItem.js:52]: Maybe add some tests for focused CheckboxItem.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: tests/screenshot/apps/components/Checkbox.js [hunk 4]
```diff
@@ -1,9 +1,20 @@
 import Checkbox from '../../../../Checkbox';
 
+import {withConfig} from './utils';
+
 const CheckboxTests = [
 	<Checkbox />,
 	<Checkbox selected />,
-	<Checkbox>star</Checkbox>,
```
Comment (dongsuwon) [inline tests/screenshot/apps/components/Checkbox.js:6]: This test seems necessary.

**Comment by dongsuwon** [comment]
LGTM

**Pull request merged**